litestar/contrib/sqlalchemy/repository/_sync.py

```diff
@@ -715,8 +715,9 @@ def check_health(cls, session: Session) -> bool:
         Returns:
             `True` if healthy.
         """
+        sql = "SELECT 1 FROM DUAL" if session.bind and session.bind.dialect.name == "oracle" else "SELECT 1"
```

Comment:
Let's please refactor this into a separate function.

Reply:
@sobolevn Thanks for the review.  Take a look and see if the latest change is what you had in mind.